pkg/apis/migration/v1alpha1/mighook_types.go

```diff
@@ -26,7 +26,8 @@ import (
 )
 
 const (
-	PhaseLabel           = "phase"
+	HookPhaseLabel       = "phase"
+	HookOwnerLabel       = "owner"
```

Comment:
I missed this on initial review, and we don't implement this across the project right now, but in the future I think we should consider following k8s guidelines for label names (e.g. including `migration.openshift.io` namespace in label name)

https://kubernetes.io/docs/concepts/overview/working-with-objects/common-labels/#labels